routemaster/models/subscriber.rb

```diff
@@ -79,6 +79,28 @@ def uuid
       _attributes['uuid']
     end
 
+    def health_points
+      _attributes.fetch('health_points', '100').to_i
+    end
+
+    def change_health_by(offset)
+      new_value = _redis_lua_run(
+        'subscriber_change_health_by',
+        keys: [_key],
+        argv: [offset]
+      )
+      @_attributes = nil
+      new_value
+    end
+
+    def last_attempted_at
+      _attributes['last_attempted_at']&.to_i
```

Comment:
This _happens_ to return zero (= origin timestamp, unix epoch) when there were no attempts, which happens to work in the rest of the code (no attempts yet = the last attempt was forever ago), but this behaviour might deserve a comment?